vuepress/docs/next/docs/getting-started/concepts-overview.md

```diff
@@ -52,14 +52,14 @@ The purpose of this guide is to give an overview of the members of an Entando cl
 
 ### Architecture diagram
 
-Let’s start with a picture of an Entando 6 cluster and how the various
+Let’s start with a picture of an Entando cluster and how the various
 members interact with each other.
 
-![Entando6 cluster architecture diagram](./img/entando-architecture-v1.3.svg)
+![Entando Cluster Architecture Diagram](./img/entando-architecture-v1.5.png)
```

Comment:
Could we switch to the svg from drawio? That tends to scale a lot better whereas png is a lossy format and makes text harder to read at smaller sizes. For diagrams especially, svg is preferred. https://www.diagrams.net/blog/export-svg